onnxruntime/core/providers/openvino/openvino_provider_factory.cc

```diff
@@ -233,6 +233,10 @@ static void ParseProviderInfo(const ProviderOptions& provider_options,
 
   pi.precision = OpenVINOParserUtils::ParsePrecision(provider_options, pi.device_type, "precision");
 
+  if (provider_options.contains("reshape_input")) {
```

Comment:
@preetha-intel  is this correct in 
 // Always true for NPU plugin or when passed .
  if (pi.device_type.find("NPU") != std::string::npos) {
    pi.disable_dynamic_shapes = true;
  }
https://github.com/intel/onnxruntime/blob/f390eb5ad1ad96a7ba3da62c0804913114ec36f9/onnxruntime/core/providers/openvino/openvino_provider_factory.cc#L350C2-L353C4